### PR TITLE
Lab2 implemented

### PR DESCRIPTION
**Pull request by mcflydesigner**

### CODE REVIEW

File: app_python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM python:3.10.4-alpine as base
+
+RUN apk add --update build-base=0.5-r3
+
+RUN pip install --upgrade pip
+
+COPY ./requirements.txt .
```
Comment (Khalil19-99) [inline app_python/Dockerfile:7]: It is better to not put files in root dir without custom user owner

**Comment by Khalil19-99** [comment]
otherwise it's all good!

File: app_python/Dockerfile (hunk 1 above)
Comment (e2xen) [reply app_python/Dockerfile:7]: +1, use `WORKDIR`

File: app_python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+RUN adduser -D app_user && chown -R app_user /usr/src/app
+USER app_user:app_user
+
+COPY --from=base /usr/local/lib/python3.10/site-packages/ /usr/local/lib/python3.10/site-packages/
+COPY --from=base /usr/local/bin/ /usr/local/bin/
+
+COPY --chown=app_user:app_user . .
+
+RUN python manage.py flush --no-input
+RUN python manage.py migrate
```
Comment (e2xen) [inline app_python/Dockerfile:27]: Combine adjacent `RUN` commands with `&&` to reduce container layers

**Comment by e2xen** [comment]
Great job

**Pull request closed without merging**